README.md

```diff
@@ -452,40 +464,84 @@ data:
   tokenurl: "<auth_url>"
   tokenurlsuffix: "/oauth/token"
 ```
+with TLS configurations:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sap-btp-service-operator-tls
+  namespace: <namespace>
+type: kubernetes.io/tls
+data:
+  tls.crt: <crt> #base64 encoded
+  tls.key: <key> #base64 encoded
+```
+
+#### Centrally-Managed Namespace Secret
+
+Define a secret named: `<namespace>-sap-btp-service-operator` in the Centrally Managed Namespace.
+
+**Note:**
+The system's centrally-managed namespace is set by the value in `.Values.manager.management_namespace`. You can provide this value during installation (refer to step 4 in the [Setup](#setup) section). 
+If you don't specify this value, the system will use the installation namespace as the default.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <namespace>-sap-btp-service-operator
+  namespace: centrally managed namespace
+type: Opaque
+data:
+  clientid: "<clientid>"
+  clientsecret: "<clientsecret>"
+  sm_url: "<sm_url>"
+  tokenurl: "<auth_url>"
+  tokenurlsuffix: "/oauth/token"
+```
+### 2. Explicit Secret Configuration  
+
+You can specify on the `ServiceInstance` level to which subaccount it should be associated:
```

Comment:
"...in which subaccount it shall be created"

Reply:
done